README.md

```diff
@@ -123,6 +123,18 @@ var errors = require('@google/cloud-errors')({
 });
 ```
 
+#### Notes On Uncaught Exception Reporting
+
+It should be noted that leaving the `reportUncaughtExceptions` configuration option set to its default of `true`
+will cause the error-reporting library to attach to the `process` objects `'uncaughtException'` event which will
+result in the following side effects:
+
+  1. Uncaught exceptions in your script will **not** be printed to `stderr` unless ones application code has itself
```

Comment:
I think you had written a small example of how this could be done in the original issue. Maybe it could be included here?